### ISSUE TITLE
Error in seq.default(idxDEL[kk[i]] + 1, idxDEL[kk[i]] + CNVdistance -  :    'from' must be a finite number

### ISSUE THREAD

**Issue by SamuelGreenrod** [robinjugas/CNproScan#1]
I keep getting this message when I try to run the program in R: 

Error in seq.default(idxDEL[kk[i]] + 1, idxDEL[kk[i]] + CNVdistance -  : 
  'from' must be a finite number

I've tried looking for a solution but I'm not too sure how to fix it. Is this a problem with my data or is the programme loading my data in the wrong format perhaps? Thank you!

**Comment by robinjugas** [comment]
Hello, thank you for trying the CNproScan. 
I'll look into that. It's prolly some untreated condition. Would it be possible to somehow share the data and command used?
I tested it on my dataset and I didnt run into anything. 
Robin Jugas

**Comment by SamuelGreenrod** [comment]
I've just sent an email to the email address on the paper (jugas@feec.vutbr.cz) with the data. If you don't use the email any more please let me know on here! Thanks.

**Comment by robinjugas** [comment]
I got the mail, will check and try what's wrong there.

**Comment by robinjugas** [comment]
Hi, so I found the error and corrected it. 
Though, I have bad news - it doesnt support multiple chromosome/contigs at the moment. You will have to split it, I'll attach scripts for that here and into repo. You will need bam index too. 
[scripts.zip](https://github.com/robinjugas/CNproScan/files/6861006/scripts.zip)
[bam_index.zip](https://github.com/robinjugas/CNproScan/files/6861011/bam_index.zip)

I had succes with this:
```
devtools::install_github("robinjugas/CNproScan",force = TRUE,ref = "main")

library(CNproScan)

setwd("/media/rj/TOSHIBA_500GB/CNproScan error")

coverage_file<-"cluster_001_consensus.coverage"
bam_file<-"Reads_mapped.bam"
fasta_file<-"cluster_001_consensus.fasta"

DF<-CNproScanCNV(coverage_file, bam_file, fasta_file,8)
```
Prior to installing the package, I think it will be useful to delete the folder likely here:   /home/$USER/R/x86_64-pc-linux-gnu-library/Rversion/CNproScan
The error persisted few times even after I updated the repo, but then it worked. 
I got 26 deletions and 7 Tandem duplications from the first contig.
